slices/gearman-job-server.yaml

```diff
@@ -0,0 +1,28 @@
+package: gearman-job-server
+
+essential:
+  - gearman-job-server_copyright
+
+slices:
+  libs:
+    essential:
+      - libboost-program-options1.83.0_libs
+      - libc6_libs
+      - libevent-2.1-7t64_libs
+      - libevent-pthreads-2.1-7t64_libs
+      - libgcc-s1_libs
+      - libhiredis1.1.0_libs
+      - libmemcached11t64_libs
+      - libmysqlclient21_libs
+      - libpq5_libs
+      - libsqlite3-0_libs
+      - libstdc++6_libs
+    contents: 
```

Comment:
Let's match style of other SDFs (Slice Definition Files) and include an empty line between sections.
I.e.:
```
      - libstdc++6_libs

    contents: 
```